specs/accounting_reform/beacon-chain.md

```diff
@@ -81,6 +81,8 @@ class BeaconState(Container):
     # Current and previous epoch participation flags
     previous_epoch_flags: List[Bitvector[8], VALIDATOR_REGISTRY_LIMIT]
     current_epoch_flags: List[Bitvector[8], VALIDATOR_REGISTRY_LIMIT]
+    # Denominator to real-time-updated balances (NOT effective balances!)
+    balance_denominator: uint64
```

Comment:
This means balances is not denoted in `GWei` anymore, should be changed to `uint64`